scripts/gtest/1.8.0-cxx11abi/.travis.yml

```diff
@@ -0,0 +1,25 @@
+language: cpp
+
+matrix:
+  include:
+    - os: linux
+      sudo: true
+      addons:
+        apt:
+          sources:
+           - ubuntu-toolchain-r-test
+          packages:
+           - g++-5
+           - libstdc++-5-dev
+
+script:
+- sudo perl -i -p -e "s/# define _GLIBCXX_USE_DUAL_ABI 0/# define _GLIBCXX_USE_DUAL_ABI 1/g;" /usr/include/x86_64-linux-gnu/c++/5/bits/c++config.h
+- export CXX=g++-5
+- export CC=gcc-5
+- ./mason build ${MASON_NAME} ${MASON_VERSION}
+
+after_success:
+- ./mason publish ${MASON_NAME} ${MASON_VERSION}
+
+install:
+- true
```

Comment:
please remove this and default to the clang++ compiler, per our other c++abi build (libprogram_options)